Fix dot notation

vaporting opened the pull request:
Dot notation with identifier name(keywords, let) don't need whitespace.

Test case 1

Before:
```
y.let .let = 1.23;
```
After:
```
y.let.let = 123;
```
Test case 2

Before:
```
y.await.break .const .delete . else .return .new .yield = 1.23;
```
After:
```
y.await.break.const.delete.else.return.new.yield = 1.23;
```

vaporting commented:
@fitzgen Hello 
Do you have a time to check my code
thank you

fitzgen commented:
Hi @vaporting, I'm not maintaining this library anymore, but I think @jasonLaster can help out, or find someone who can.

jasonLaster commented:
Thanks. I'll take a look

darkwing commented:
This looks good @vaporting  :) . The only problem is your new `if` code is throwing lint errors:

```
pretty-fast.js: line 333, col 6, Error - Expected { after 'if' condition. (curly)
pretty-fast.js: line 336, col 6, Error - Expected { after 'if' condition. (curly)
pretty-fast.js: line 339, col 6, Error - Expected { after 'if' condition. (curly)
```

Could you change your `if` code to something like:

```
var ttl = token.type.label;
if (ltt == "name" && ttl == "name") {
    return true;
}
if (ltt == "name" && ttl == "num") {
    return true;
}
if (ltt == "name" && token.type.keyword) {
    return true;
}
```

Thank you!

vaporting commented:
Thanks you @darkwing 
I fix the ```if``` format problem
Could you try this new version
Thanks you!

darkwing commented:
Could you move the `if` to just one line?  It's length is still well under our linting number.  Thank you!

mozbugzi2 commented:
I would recommend waiting until #22 is merged, because there is some overlap between the patches.

jasonLaster commented:
@vaporting could you rebase?

vaporting commented:
Hi @jasonLaster,
Thank you for notifying me that there have conflicts with master branch.
I have fixed the conflicts and modified some code for the same issue.
I ran the test.js and this modification passes all tests.
Could you help me to check the code and accept this merge request?
Thank you.

jasonLaster commented:
Thanks for the help @vaporting
